resources/views/bread/browse.blade.php

```diff
@@ -87,7 +87,8 @@
                                         @endcan
                                         @foreach($dataType->browseRows as $row)
                                             <td>
-                                                <?php $options = json_decode($row->details); ?>
+                                                <?php $options = ($row->details) ? json_decode
```

Comment:
@fletch3555 it better we have the some accessor on the model level so that we dont need to hack around on views. Data Validation on Presentation layer seems like patch work.